### PR TITLE
SUMO-136259: Unified Monitors Terraform Support (BETA)

### PR DESCRIPTION
**Pull request by rsanbhad**
Initial support for the new Unified Monitors API, which will be in closed beta starting September 6 (09/06/2020). Putting the PR up for early/initial review so it can be merged when we go beta.

### CODE REVIEW

File: sumologic/resource_sumologic_monitors_library_monitor.go [hunk 1]
```diff
@@ -0,0 +1,306 @@
+// ----------------------------------------------------------------------------
```
Comment (rsanbhad) [inline sumologic/resource_sumologic_monitors_library_monitor.go:1]: This isn't really true anymore - can be removed, as the provider will most likely need to be maintained by hand until v2 of unified monitors.

File: sumologic/sumologic_client.go [hunk 2]
```diff
@@ -25,13 +25,15 @@ type Client struct {
...
 var endpoints = map[string]string{
-	"us1": "https://api.sumologic.com/api/",
-	"us2": "https://api.us2.sumologic.com/api/",
-	"eu":  "https://api.eu.sumologic.com/api/",
-	"au":  "https://api.au.sumologic.com/api/",
-	"de":  "https://api.de.sumologic.com/api/",
-	"jp":  "https://api.jp.sumologic.com/api/",
-	"ca":  "https://api.ca.sumologic.com/api/",
+	"penguin": "https://penguin-api.ephemeral.in/api/",
+	"nite":    "https://nite-api.sumologic.net/api/",
```
Comment (seansain) [inline sumologic/sumologic_client.go:29]: we shouldn't leave these in here

**Comment by seansain** [comment]
Can you also open this PR from within the repo instead of your fork?

**Comment by rsanbhad** [comment]
Closing and opening new PR from within the repo as requested

**Pull request closed without merging**